code/modules/mob/living/carbon/human/life/handle_status_effects.dm

```diff
@@ -9,32 +9,33 @@
 
 	//Periodically double-check embedded_flag
 	if(embedded_flag && !(life_tick % 10))
-		var/list/E
-		E = get_visible_implants(0)
-		if(!E.len)
-			embedded_flag = 0
+		var/list/L = get_visible_implants(0)
+		if(!length(L))
```

Comment:
if !length(get_visible_implants(0))

Reply:
but really you should remove this completely and just have the embedded removal code check it since this is meant to be an 'optimisation' pr

Reply:
Could you cut out the passive aggressive attitude please?

Reply:
i will if you stop acting like everything i say is passive aggressive (nothing in this comment thread is passive aggressive, gasp)

Reply:
"inb4 lak just ignores all my comments."
"since this is meant to be an 'optimisation' pr"
"(nothing in this comment thread is passive aggressive, gasp)"
These are all passive-aggressive. Try to maintain a neutral tone when reviewing PRs.